js/main.js

```diff
@@ -1,10 +1,218 @@
 const menuButton = document.getElementById('destop-menu');
 const mainElement = document.getElementById('section1');
+const workSection = document.getElementById('portfolio');
 const navId = document.getElementById('navId');
 const menuIcon = document.getElementById('hm-menuIcon');
 const logoText = document.getElementById('logo-text');
 const mobileNavigation = document.querySelectorAll('.clicked-item');
 
+const modalClass = document.getElementById('modalClass');
+
+const work1 = {
+  name: 'Tonic',
+  description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  featuredImage: 'image/works/SnapshootPortfolio.svg',
+  technologies: ['html', 'css', 'javascript', 'bootstrap'],
+  linkToLiveVersion: 'liveversion.com',
+  linkToSource: 'github.com/bhskr44',
+};
+
+const work2 = {
+  name: 'Multi-Post Stories',
+  description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  featuredImage: 'image/works/SnapshootPortfolio2.svg',
+  technologies: ['html', 'css', 'javascript'],
+  linkToLiveVersion: 'liveversion.com',
+  linkToSource: 'github.com/bhskr44',
+};
+
+const work3 = {
+  name: 'Tonic',
+  description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  featuredImage: 'image/works/SnapshootPortfolio3.svg',
+  technologies: ['html', 'css', 'javascript'],
+  linkToLiveVersion: 'liveversion.com',
+  linkToSource: 'github.com/bhskr44',
+};
+const work4 = {
+  name: 'Multi-Post Stories',
+  description:
+    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+  featuredImage: 'image/works/SnapshootPortfolio4.svg',
+  technologies: ['html', 'css', 'javascript'],
+  linkToLiveVersion: 'liveversion.com',
+  linkToSource: 'github.com/bhskr44',
+};
```

Comment:
- [ ]Good job so far :+1:  I would like to recommend you refactor your code to make it more clear you need to create an array of objects, the objects represent each project you need to store the following pieces of data  instead of creating the object and after creating the array , it can be confusing :smile:  Please fix it, you can check the example below :arrow_down: 
```
const projects = [
  {
  projectId: 'work1',
  name: '1 Tonic',
  description:
    'A daily selection of privately personalized reads; no accounts or sign-ups required.',
  featuredImage: 'image/works/SnapshootPortfolio.svg',
  technologies: ['html', 'css', 'javascript', 'bootstrap'],
  linkToLiveVersion: 'liveversion.com',
  linkToSource: 'github.com/bhskr44',
};
{
projectId: 'work2',
....
};
{
projectId: 'work3',
....
}

]
````